cycle_for/task_05.py

```diff
@@ -0,0 +1,9 @@
+# Дано несколько чисел. Вычислите их сумму. Сначала вводите количество чисел N, затем вводится ровно N целых чисел.
+# Какое наименьшее число переменных нужно для решения этой задачи?
```

Comment:
Same as last question. You're using way too many variables here. At every iteration, you're creating a new variable `number`

Reply:
```
N = int(input())
sum = 0
for number in range(N):
    sum += int(input())
print(sum)
```